content/api/event-alert/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Event & alert API"
+title: "Event & Alert"
```

Comment:
Sentence case

Reply:
Was readability the only concern here? I agree with the content (including headers) being sentence case but feels title is better of being title case.

Reply:
Also I was referring to Vault docs and they were using title case for headers as well and it doesn't affect the readability (may be its just me)
https://www.vaultproject.io/api-docs/secret/kv/kv-v2

Reply:
Capitalization is used inconsistently in the docs. Enforcing sentence case minimizes the number of rules and exceptions as a way of improving the consistency and quality of the docs.

Reply:
Ok, I will need to get used to it